js/config/local.config.js

```diff
@@ -0,0 +1,9 @@
+define([], function () {
+	var config = {};
+	config.userAuthenticationEnabled = true;
```

Comment:
This was disabled by default. Same as props below. Why was the behavior changed?